test/functional/page_objects/common_page.js

```diff
@@ -270,6 +270,19 @@ export function CommonPageProvider({ getService, getPageObjects, getPageObject }
     async isChromeVisible() {
       return await testSubjects.exists('kibanaChrome');
     }
+
+    async isTopNavVisible() {
+      return await testSubjects.exists('top-nav');
+    }
+
+    async waitForTopNavToBeVisible() {
+      await retry.try(async () => {
+        const isNavVisible = await this.isTopNavVisible();
+        if (!isNavVisible) {
+          throw 'Local nav not visible yet';
```

Comment:
Unless we're handling errors in a weird way in the new functional test runner that makes throwing strings, can you throw this message inside a `new Error()` so that it comes with a stack trace and such?